lib/bolt/cli.rb

```diff
@@ -159,25 +159,22 @@ def parse_command
     # Loads the project and configuration. All errors that are raised here are not
     # handled by the outputter, as it relies on config being loaded.
     def load_config
-      @config = if ENV['BOLT_PROJECT']
-                  project = Bolt::Project.create_project(ENV['BOLT_PROJECT'], 'environment')
-                  Bolt::Config.from_project(project, options)
-                elsif options[:configfile]
-                  Bolt::Config.from_file(options[:configfile], options)
+      project = if ENV['BOLT_PROJECT']
+                  Bolt::Project.create_project(ENV['BOLT_PROJECT'], 'environment')
                 else
                   cli_flag = options[:project] || options[:boltdir]
```

Comment:
`options[:boltdir]` can't be set anymore, this can be inlined to the `if` below.